README.md

```diff
@@ -1,2 +1,175 @@
-# ruby-capstone-kanban
-This repository is created to setup kanban board
+https://github.com/Xghayor/ruby-group-capstone.git
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ruby-group-capstone] <a name="about-project"></a>
+
+**ruby-group-capstone** Is a console app that helps keep records of different types of things: **books**, **music albums**, and **games**.The data is stored in JSON files. A database with tables structure analogical to the program's class structure is prepared.
+
+## Video Presentation [https://drive.google.com/file/d/1r927HLGVXW3egeSP1Xii7LSXzLILJXNf/view?usp=sharing]
```

Comment:
[OPTIONAL] I think you should make the link a small text instead of making it a large text along with the video presentation subheading
[REQUIRED] Good job adding a video presentation of your project but your video doesn't fulfill some requirements:
- Kindly ensure that all members of your team are present in the video recording and that everyone presents something
- Give a clear and objective description of the project in good English'
- The video should not exceed 5 minutes
- Kindly point out a piece of code you find interesting
-